### ISSUE TITLE
Renaming theme

### ISSUE THREAD

**Issue by SamsamTS** [boformer/BuildingThemes#57]
I think renaming a theme is not possible with the current system.

In the districts configuration, the names of enabled themes are saved. If the name of a theme is changed later on, the association is lost unless we update the name in every district configuration for all possible saves. Which can prove to be problematic.

One possible solution would be to generate a unique ID. This unique ID would never change for a given theme. That id would be saved instead of the name.

Of course changing this now would break the current saved districts. So maybe we could keep the name but add a "displayName" property to the theme. It could be changed at will while the internal name doesn't ever change. If the displayName doesn't exist, the internal name is displayed instead.

Or I can ditch the rename idea altogether.

**Comment by boformer** [comment]
Yes, also thought about a display name property. I think it is the best solution.

I think renaming themes is a low-priority feature. It is something we can always add later.

**Comment by bloodypenguin** [comment]
I think a guid should be assigned to each theme to serve as the identifier

**Comment by boformer** [comment]
What would be the benefit of that? Name conflicts are extremely rare.
